mne/viz/evoked.py

```diff
@@ -345,7 +345,7 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
                     # plot channels
                     for ch_idx, z in enumerate(z_ord):
                         line_list.append(ax.plot(times, D[ch_idx], picker=3.,
-                                                 zorder=1 + z,
+                                                 zorder=z + 1 if spatial_colors else 1,
```

Comment:
I think by default lines get `zorder=2` in matplotlib, can you check? Or do you mean to set the zorder lower?